crates/holochain_zome_types/src/query.rs

```diff
@@ -61,7 +61,8 @@ impl Default for ChainQueryFilterRange {
 #[derive(
     serde::Serialize, serde::Deserialize, SerializedBytes, Default, PartialEq, Clone, Debug,
 )]
-#[non_exhaustive]
+// TODO: get feedback on whether it's OK to remove non_exhaustive
+// #[non_exhaustive]
```

Comment:
I removed non_exhaustive just because I was lazy and wanted it to be easier to write my test. Also I don't think our API is stable enough to warrant niceties like this yet. I will put it back in if anyone wants it.

Reply:
If we ever want this, it'll be easier to include it sooner, specifically because of things like tests getting written without it. But I don't have an opinion otherwise.